roaster-tm4j/roaster-tm4j-common/src/main/java/hu/icellmobilsoft/roaster/tm4j/common/config/InvalidConfigException.java

```diff
@@ -0,0 +1,44 @@
+/*-
+ * #%L
+ * Coffee
+ * %%
+ * Copyright (C) 2020 i-Cell Mobilsoft Zrt.
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+package hu.icellmobilsoft.roaster.tm4j.common.config;
+
+/**
+ * Exception class indicating errors regarding the Roaster TM4J configuration.
+ *
+ * @author martin.nagy
+ * @since 0.2.0
+ */
+public class InvalidConfigException extends RuntimeException {
```

Comment:
public class InvalidConfigException extends TestException {

egyebkent ha mar tobb exception is keletkezik akkor kell a package sajat os es akkor abbl szarmazzanak